File: Modul 4/Bahasa C (modul 4)/PRAK405_2310817320015_AlfisyahRinaFajriati.c
#include <stdio.h>
int main(){
    printf("Input\n");
    int bil1, bil2, a, b, x, y, z, hasil, total ;
    scanf("%d %d", &bil1, &bil2);
    
    printf("\nOutput\n");
    for (a = 1; a <= bil1; a++){
        for (b = a; b > 1; b--){
            printf("(%d * %d) + ", b, bil2); }
        for (x = 1, hasil = x * bil2; x < a; x++, hasil += (x * bil2)){}
        printf("(%d * %d) = %d\n", b, bil2, hasil); }
    for (y = 1, z = 1, total = 0; y <= bil1; z += y + 1,  y++) {
        total += z * bil2 ; }
    printf("%d", total);

    return 0;
}

File: Modul 4/Bahasa Python (modul 4)/PRAK405-2310817320015-AlfisyahRinaFajriati.py
print("Input")
bil1, bil2 = input().split()
bil1 = int(bil1) ; bil2 = int(bil2)

print("\nOutput")
for i in range(1, bil1 + 1) :
    j = i 
    while j > 1 :
        print("(%d * %d) + "%(j, bil2), end='') ; j -= 1
    x = 1 ; hasil = (i * bil2)
    while x < i :
        hasil += (x * bil2) ; x += 1
    print("(%d * %d) = %d"%(j, bil2, hasil))
y = 1 ; z = 1 ; total = 0
while y <= bil1 :
    total += z * bil2 ; z += y + 1 ; y += 1
print("%d"%(total))